site/search/Search.tsx

```diff
@@ -95,7 +98,7 @@ export const Search = ({
                     <SearchResultTypeToggle />
                 </div>
                 <div className="search-template-results col-start-2 span-cols-12">
-                    <SearchNoResults />
+                    {!isFetching && <SearchNoResults />}
```

Comment:
This is not strictly necessary when everything else is working correctly, but I briefly had a bug related to rendering skeletons that meant we showed "No results" intermittently when data was loading, which we should never do.